app/views/tasks/_model_solution_form.html.slim

```diff
@@ -6,7 +6,7 @@
   = model_solution.text_field :internal_description, class: 'form-control'
 .field-element
   = model_solution.label :xml_id, t('tasks.tests.xml_id'), class: 'form-label'
-  = model_solution.text_field :xml_id, class: 'form-control'
+  = model_solution.text_field :xml_id, class: 'form-control', value: test.object.xml_id.presence || '__nested_field_for_replace_with_index__'
```

Comment:
That is using a [magic string](https://github.com/ncri/nested_form_fields#index-replacement-string), right? Looks good, but wasn't aware of this feature :)